ForkMe-Mobile/app/src/main/res/layout/activity_user_repositories_view.xml

```diff
@@ -0,0 +1,176 @@
+    android:background="@color/colorWhite"
+    tools:context="dlei.forkme.gui.activities.github.UserRepositoriesViewActivity">
+
+    <include layout="@layout/nav_drawer_toolbar"
+        android:id="@+id/toolbar_layout"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        android:layout_height="58dp"
+        android:layout_width="0dp" />
+
+
+
+    <include layout="@layout/progress_bar_spinner"
+        android:id="@+id/progress_bar_spinner_layout"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintLeft_toLeftOf="parent"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_centerHorizontal="true"
+        />
+
+    <View
+        android:layout_width="0dp"
+        android:layout_height="1dp"
+        android:background="?android:attr/dividerHorizontal"
+        android:id="@+id/viewHeaderSeparator"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintHorizontal_bias="0.0"
+        android:layout_marginTop="@dimen/padding_extra"
+        app:layout_constraintTop_toBottomOf="@+id/repositoryCountText" />
+
+    <android.support.v7.widget.RecyclerView
+        android:id="@+id/repositoryRecyclerView"
+        android:orientation="vertical"
+        android:layout_width="match_parent"
+        android:layout_height="320dp"
+        android:layout_marginEnd="1dp"
+        android:layout_marginLeft="1dp"
+        android:layout_marginRight="1dp"
+        android:layout_marginStart="1dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintHorizontal_bias="0.965"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        android:layout_marginTop="8dp"
+        app:layout_constraintTop_toBottomOf="@+id/viewHeaderSeparator"
+        app:layout_constraintVertical_bias="0.0" />
+
+    <android.support.v7.widget.AppCompatImageView
+        android:id="@+id/userAvatarIconImage"
+        android:layout_width="48dp"
+        android:layout_height="48dp"
+        android:layout_marginLeft="@dimen/padding_extra"
+        android:layout_marginStart="@dimen/padding_extra"
+        android:layout_marginTop="@dimen/padding_extra"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/toolbar_layout"
+        app:srcCompat="@mipmap/forkme_clear_launcher"
+        app:layout_constraintRight_toLeftOf="@+id/userBioText"
+        android:layout_marginRight="8dp" />
+
+    <android.support.v7.widget.AppCompatTextView
```

Comment:
["This will automatically be used when you use TextView in your layouts. You should only need to manually use this class when writing custom views."](https://developer.android.com/reference/android/support/v7/widget/AppCompatTextView.html)